### PR TITLE
Update for Feb 2024 Collaboration meeting

### PR DESCRIPTION
**Pull request by JoanneBogart**
Update index.html for Feb 2024 Collaboration meeting.  Archive Oct 2023  Sprint Week version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,251 +2,240 @@
 <html>
 
 <head>
-     <title>Checklist for the October 2023 LSST DESC Sprint Week @ CMU</title>
+     <title>Checklist for the February 2023 LSST DESC Virtual Collaboration Meeting</title>
```
Comment (yymao) [inline index.html]:
```suggestion
     <title>Checklist for the February 2024 LSST DESC Virtual Collaboration Meeting</title>
```

File: index.html [hunk 2]
```diff
@@ -2,251 +2,240 @@
...
+     </div>
+     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+     <!-- Calendar -->
+     <div class="item">
+          <div class="box">
+               <input type="checkbox">
+          </div>
+          <div class="text">
+               Check out the
+               <a href="https://teamup.com/kshoa4w4symu54dfke">
```
Comment (yymao) [inline index.html]:
```suggestion
               <a href="https://teamup.com/kshoa4w4symu54dfke"
```

**Review by yymao — CHANGES_REQUESTED** [review]
Caught two minor typos.

**Comment by JoanneBogart** [comment]
Oops!   Thank you, @yymao !

**Review by yymao — APPROVED** [review]
Thanks! LGTM

**Pull request merged**